### DISCUSSION TITLE
Operating system

### DISCUSSION THREAD

**Discussion by HarryC1998** [netbox-community/netbox#10625] [General]
Hi All, 

I am a bit new to netbox. 

Is there anyway of inputting operating systems fields for servers?

**Comment by hagbarddenstore** [comment]
Hi and welcome!

You can use the Platform field on Device or Virtual Machine to represent the operating system.

Remember to *not* set the Manufacturer field on the Platform. Manufacturer should only be set on a Platform if that Platform is tied to a specific Device manufacturer. Example is Cisco IOS for Cisco devices.

**Reply by HarryC1998** [reply]
thanks. I thought it would be the platform field. Thanks again for the quick Reponses.
